docs/src/main/asciidoc/README.adoc

```diff
@@ -6,6 +6,27 @@ include::intro.adoc[]
 
 > NOTE: if you are looking for a way to bind to services then this is the wrong library. Check out the [Spring Cloud Connectors](https://github.com/spring-cloud/spring-cloud-connectors) instead.
 
+= Building documentation
```

Comment:
Good point. Is this PR under consideration? I'll be happy to add the above if so.